src/Console/Command/ReleaseCommand.php

```diff
@@ -0,0 +1,318 @@
+    );
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        parent::configure();
+
+        $help = <<<'EOT'
+The <info>release</info> command analyzes pull request of a given project to determine
+the changelog and the next version to release.
+
+Usage:
+
+<info>php dev-kit release</info>
+
+First, a question about what bundle to release will be shown, this will be autocompleted will
+the projects configured on <info>projects.yml</info>
+
+The command will show what is the status of the project, then a list of pull requests
+made against selected branch (default: stable branch) with the following information:
+
+stability, name, labels, changelog, url.
+
+After that, it will show what is the next version to release and the changelog for that release.
+EOT;
+
+        $this
+            ->setName('release')
+            ->setDescription('Helps with a project release.')
+            ->setHelp($help);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $project = $this->getProject($input, $output);
+        $branches = array_keys($this->configs['projects'][$project]['branches']);
+        $branch = count($branches) > 1 ? next($branches) : current($branches);
+
+        try {
+            $package = $this->packagistClient->get(static::PACKAGIST_GROUP.'/'.$project);
+            $this->io->title($package->getName());
+            $this->prepareRelease($package, $branch, $output);
+        } catch (ExceptionInterface $e) {
+            $this->io->error('Failed with message: '.$e->getMessage());
+        }
+
+        return 0;
+    }
+
+    private function getProject(InputInterface $input, OutputInterface $output)
+    {
+        $helper = $this->getHelper('question');
+
+        $question = new Question('<info>Please enter the name of the project to release:</info> ');
+        $question->setAutocompleterValues(array_keys($this->configs['projects']));
+        $question->setNormalizer(function ($answer) {
+            return $answer ? trim($answer) : '';
+        });
+        $question->setValidator(function ($answer) {
+            if (!array_key_exists($answer, $this->configs['projects'])) {
+                throw new \RuntimeException('The name of the project should be on `projects.yml`');
+            }
+
+            return $answer;
+        });
+        $question->setMaxAttempts(3);
+
+        return $helper->ask($input, $output, $question);
+    }
+
+    private function prepareRelease(Package $package, $branch, OutputInterface $output)
+    {
+        $repositoryName = $this->getRepositoryName($package);
+
+        try {
+            $currentRelease = $this->githubClient->repo()->releases()->latest(
+                static::GITHUB_GROUP,
+                $repositoryName
+            );
+        } catch (ExceptionInterface $e) {
+            throw new RuntimeException('Current release not found');
```

Comment:
```suggestion
            throw new RuntimeException('Current release not found', 0, $e);
```

Reply:
I removed this exception to help debuggin problems, just like the other one